--- a/mingw-w64-crt/misc/output_format.c	2014-01-24 23:33:39.869904000 +0000
+++ b/mingw-w64-crt/misc/output_format.c	2014-01-24 23:33:18.281678300 +0000
@@ -1,7 +1,11 @@
+#define _get_output_format __dummy__get_output_format
+#define _set_output_format __dummy__set_output_format
 #include <windows.h>
-#include <stdio.h>
 #include <msvcrt.h>
 
+#undef _get_output_format
+#undef _set_output_format
+
 static unsigned int last_value = 0;
 typedef unsigned int (*f_get_output_format)(void);
 typedef unsigned int (*f_set_output_format)(unsigned int);
@@ -9,6 +13,12 @@
 static unsigned int init_set_output_format(unsigned int);
 f_set_output_format __MINGW_IMP_SYMBOL(_set_output_format) = init_set_output_format;
 
+unsigned int _set_output_format(unsigned int format);
+unsigned int _set_output_format(unsigned int format)
+{
+    return __MINGW_IMP_SYMBOL(_set_output_format)(format);
+}
+
 static unsigned int fake_set_output_format(unsigned int value)
 {
     return InterlockedExchange((LONG*)&last_value, value);
@@ -29,6 +39,12 @@
 static unsigned int init_get_output_format(void);
 f_get_output_format __MINGW_IMP_SYMBOL(_get_output_format) = init_get_output_format;
 
+unsigned int _get_output_format(void);
+unsigned int _get_output_format(void)
+{
+    return __MINGW_IMP_SYMBOL(_get_output_format)();
+}
+
 static unsigned int fake_get_output_format(void)
 {
     return last_value;
--- a/mingw-w64-crt/misc/lc_locale_func.c	2014-01-24 23:31:53.234142600 +0000
+++ b/mingw-w64-crt/misc/lc_locale_func.c	2014-01-24 23:31:13.630449400 +0000
@@ -4,6 +4,7 @@
 #include <locale.h>
 #include <msvcrt.h>
 
+#undef __lc_codepage
 #undef ___lc_codepage_func
 #include "mb_wc_common.h"
